packages/melos/lib/src/command_runner/format.dart

```diff
@@ -19,6 +19,11 @@ class FormatCommand extends MelosCommand {
           '[write]              Overwrite formatted files on disk.\n',
       abbr: 'o',
     );
+    argParser.addOption(
+      'line-length',
+      defaultsTo: '80',
```

Comment:
I think it's better to have this nullable and just not add the flag in the cases when this isn't passed in.

Reply:
Thank you for your feedback